Gemfile.lock

```diff
@@ -0,0 +1,65 @@
+PATH
+  remote: .
+  specs:
+    pedicel-pay (0.0.7)
+      pedicel (~> 1.1.0)
```

Comment:
```suggestion
      pedicel (~> 1.2.0)
```

I've released version 1.2.0 of pedicel based on https://github.com/clearhaus/pedicel/pull/41.